alcs-api/src/application/application-document/application-document.entity.ts

```diff
@@ -0,0 +1,36 @@
+import { AutoMap } from '@automapper/classes';
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { Document } from '../../document/document.entity';
+import { Application } from '../application.entity';
+
+@Entity()
+export class ApplicationDocument extends BaseEntity {
+  constructor(data?: Partial<ApplicationDocument>) {
+    super();
+    if (data) {
+      Object.assign(this, data);
+    }
+  }
+
+  @AutoMap()
+  @PrimaryGeneratedColumn('uuid')
+  uuid: string;
+
+  @Column()
+  type: 'decisionDocument';
```

Comment:
why not default value?

Reply:
I want to avoid having to do a migration when we add a second type of document.